HyperPlay-Gaming/hyperplay-desktop-client
# feat(quest): add onchain reward confirmation modal

closes https://github.com/HyperPlay-Gaming/product-management/issues/532

how to test:

1. try to claim an on-chain request
2. check that the confirmation modal is shown
3. check that the claim flow is resumed after hitting confirm

---

Use the following Checklist if you have changed something on the Backend or Frontend:
